(F841) Local variable name is assigned to but never used

| | |
|:---|:---|
|**_F´ Version_**| devel |
|**_Affected Component_**| string_util.py  |
---
## Problem Description

Ruff linter spotted one issue in:

https://github.com/fprime-community/fprime-tools/blob/58e28dfa4a1c93e55aff533747def2b5386aa40f/src/fprime/util/string_util.py#L95

```
src/fprime/util/string_util.py:95:26: F841 [*] Local variable `e` is assigned to but never used
```

According to Flake 8: "A local variable in your function was defined but is not being used. This local variable should be removed."

Ref: https://www.flake8rules.com/rules/F841.html

## Expected Behavior

The variable should be used or be removed.

PS: can be tagged as `Good First Issue`

--- a/src/fprime/util/string_util.py
+++ b/src/fprime/util/string_util.py
@@ -92,7 +92,7 @@ def format_string_template(format_str, given_values):
         result = formatted_str.format(*values)
         result = result.replace("%%", "%")
         return result
-    except ValueError as e:
+    except ValueError:
         msg = "Value and format string do not match. "
         msg += " Will ignore integer flags `d` in string template. "
         msg += f"values: {values}. "

